### PR TITLE
[SHELL32] shlexec: Simplify parameters parsing

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

Simplify code logic.
JIRA issue: N/A

## Proposed changes

- Use `PathGetArgsW`, `PathRemoveArgsW`, and `PathUnquoteSpacesW` for parsing parameters.

### CODE REVIEW

File: dll/win32/shell32/shlexec.cpp
Comment (HBelusca) [inline dll/win32/shell32/shlexec.cpp:2185]: Not sure about that `parametersLen` setting.

Comment (katahiromz) [reply dll/win32/shell32/shlexec.cpp:2185]: This is buffer length that will be used later.

Comment (HBelusca) [reply dll/win32/shell32/shlexec.cpp:2185]: It doesn't appear to be used later.

**Review by HBelusca — APPROVED** [review]
LGTM but let's wait for another review(@whindsaks perhaps?) before merging.

File: dll/win32/shell32/shlexec.cpp
Comment (katahiromz) [reply dll/win32/shell32/shlexec.cpp:2185]:
```suggestion
```

**Pull request merged**
